tests/functional/pv/test_qos_limits.py

```
@@ -0,0 +1,568 @@
+            "metadata": {"name": self.gold_vac_name},
+            "driverName": "openshift-storage.rbd.csi.ceph.com",
+            "parameters": {
+                "maxReadBps": self.gold_limits["rbps"],
+                "maxWriteBps": self.gold_limits["wbps"],
+                "maxReadIops": self.gold_limits["riops"],
+                "maxWriteIops": self.gold_limits["wiops"],
+            },
+        }
+
+        log.info("Writing clean VolumeAttributesClass payload manifests...")
+        tmp_silver = f"/tmp/{self.silver_vac_name}.json"
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)